front/components/vaults/VaultResourcesList.tsx

```diff
@@ -167,9 +167,7 @@ const getTableColumns = ({
             !info.row.original.dataSourceView.dataSource
               .fetchConnectorError && <Chip color="amber">Never</Chip>}
           {info.row.original.dataSourceView.dataSource.fetchConnectorError && (
-            <Chip color="warning">
-              Error loading the connector. Try again in a few minutes.
-            </Chip>
+            <Chip color="warning">Retry in a few minutes</Chip>
```

Comment:
The proper fix should probably be in `DataTable` to have flexible columns in their, right 🙃.

Reply:
Respectfully disagree, we don't want the sync chip to be too long so the proper fix is to keep it short.

Reply:
Making a functional tooltip would be the full proper fix IMO (but tougher job)

Reply:
It's not a tooltip here but a Chip, right?

Reply:
It's a chip, and the "Try again in a few minutes" should be in a tooltip when you hover the chip

Reply:
👍 Let's merge this until we refactor the table.